src/renderers/WebGLRenderer.js

```diff
@@ -1415,7 +1415,7 @@ function WebGLRenderer( parameters = {} ) {
 
 		const fog = scene.fog;
 		const environment = material.isMeshStandardMaterial ? scene.environment : null;
-		const encoding = ( _currentRenderTarget === null ) ? _this.outputEncoding : _currentRenderTarget.texture.encoding;
+		const encoding = ( _currentRenderTarget === null ) ? _this.outputEncoding : LinearEncoding;
```

Comment:
I think we still need to consider the WebGL1 case. We still need to do inline encoding if context is WebGL1.

Reply:
Sorry, didn't realize `ext_sRGB` is required now.